lib/src/DateTimePicker/DateTimePickerToolbar.tsx

```diff
@@ -61,7 +63,14 @@ export const DateTimePickerToolbar: React.FC<ToolbarComponentProps> = ({
             />
           </Grid>
 
-          <Grid item container xs={6} justify="center" alignItems="flex-end">
+          <Grid
+            item
+            container
+            xs={6}
+            justify="center"
+            alignItems="flex-end"
+            direction={rtl ? 'row-reverse' : 'rtl'}
```

Comment:
`{rtl ? 'row-reverse' : 'row'}` ??

Reply:
Yeah, brain and fingers must have had conflicting signals there. Funny that it worked anyways.